package.json

```diff
@@ -38,6 +39,7 @@
     "cookie": ">=0.4.1",
     "ejs": ">=3.1.6",
     "escape-string-regexp": ">=2.0.0",
+    "eslint-config-standard": "^16.0.2",
```

Comment:
This looks like an accident.

Reply:
Eslint didn't work for me without this dependency. 🤷

Reply:
Still doesn't belong in the PR, and/or non dev dependencies.